.yarnrc.yml

```diff
@@ -1,6 +1,10 @@
+compressionLevel: mixed
```

Comment:
These options were added automatically by Yarn 4. As far as I can see, they disable default behavior that was turned on:

- https://yarnpkg.com/configuration/yarnrc#compressionLevel
- https://yarnpkg.com/configuration/yarnrc#enableGlobalCache